pkg/admission/handler.go

```diff
@@ -42,3 +54,32 @@ func NewHandler(ops ...Operation) *Handler {
 		operations: operations,
 	}
 }
+
+// SetReadyFunc allows late registration of a ReadyFunc to know if the handler is ready to process requests.
+func (h *Handler) SetReadyFunc(readyFunc ReadyFunc) {
```

Comment:
We have places that rely on multiple caches.  You want them unioned separately or would you be ok make this `AddReadyFunc` and waiting for all of them?